kytos/utils/users.py

```diff
@@ -0,0 +1,144 @@
+"""Module used to handle Users in Napps Server."""
+import logging
+import re
+from getpass import getpass
+
+from kytos.utils.client import UsersClient
+from kytos.utils.config import KytosConfig
+
+log = logging.getLogger(__name__)
+
```

Comment:
Alter the patterns bellow in a way that `pattern[0]` will always retrieve  the full description/explanation for that pattern, and `pattern[1]` return the regex pattern itself, otherwise you wouldn't know if you have to get `pattern[0]` or `pattern[1]` or `pattern[2]`.